### ISSUE TITLE
error when generating a seed on the latest dev version

### ISSUE THREAD

**Issue by jesse100190** [TestRunnerSRL/OoT-Randomizer#938]
when i try to generate a seed i keep getting this error using the AJEB432NSZUBAHCAE4QDASADWSDSJCMSTG62DSJULSKG5WWVW4M4TLM3XZAEV3VBFBB settings string
```
required argument is not an integer
Traceback (most recent call last):
  File "C:\Users\Jesse\OneDrive\OoT-Randomizer-Dev\OoTRandomizer.py", line 72, in start
    main(settings)
  File "C:\Users\Jesse\OneDrive\OoT-Randomizer-Dev\Main.py", line 96, in main
    return patch_and_output(settings, window, spoiler, rom, start)
  File "C:\Users\Jesse\OneDrive\OoT-Randomizer-Dev\Main.py", line 231, in patch_and_output
    patch_rom(spoiler, worlds[settings.player_num - 1], rom)
  File "C:\Users\Jesse\OneDrive\OoT-Randomizer-Dev\Patches.py", line 1275, in patch_rom
    rom.write_bytes(rom.sym('cfg_item_overrides'), get_override_table_bytes(override_table))
  File "C:\Users\Jesse\OneDrive\OoT-Randomizer-Dev\Patches.py", line 1726, in get_override_table_bytes
    return b''.join(sorted(itertools.starmap(override_struct.pack, override_table)))
struct.error: required argument is not an integer
```

**Comment by aplumafreak500** [comment]
Can confirm:
```
Traceback (most recent call last):
  File "OoTRandomizer.py", line 72, in start
    main(settings)
  File "/host-rootfs/storage/emulated/0/Android/data/tech.ula/files/OoT-Randomizer/Main.py", line 96, in main
    return patch_and_output(settings, window, spoiler, rom, start)
  File "/host-rootfs/storage/emulated/0/Android/data/tech.ula/files/OoT-Randomizer/Main.py", line 231, in patch_and_output
    patch_rom(spoiler, worlds[settings.player_num - 1], rom)
  File "/host-rootfs/storage/emulated/0/Android/data/tech.ula/files/OoT-Randomizer/Patches.py", line 1275, in patch_rom
    rom.write_bytes(rom.sym('cfg_item_overrides'), get_override_table_bytes(override_table))
  File "/host-rootfs/storage/emulated/0/Android/data/tech.ula/files/OoT-Randomizer/Patches.py", line 1726, in get_override_table_bytes
    return b''.join(sorted(itertools.starmap(override_struct.pack, override_table)))
struct.error: required argument is not an integer
```

**Comment by Zannick** [comment]
Should be fixed in 5.1.25.

**Comment by aplumafreak500** [comment]
Indeed, the error no longer occurs on my end.
